### PR TITLE
Release notes 2.8.7

### PR DESCRIPTION
**Pull request by madkinsz**
[Rendered](https://github.com/PrefectHQ/prefect/blob/release-notes-2.8.7/RELEASE-NOTES.md#release-287)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,80 @@
...
+
+- https://github.com/PrefectHQ/prefect/pull/8702
+- https://github.com/PrefectHQ/prefect/pull/8887
+- https://github.com/PrefectHQ/prefect/pull/8903
+- https://github.com/PrefectHQ/prefect/pull/8830
+
+### Results page
+
+...
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:40]:
```suggestion
See the following pull requests for implementation details:
- https://github.com/PrefectHQ/prefect/pull/8904
- https://github.com/PrefectHQ/prefect/pull/8759
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,80 @@
 # Prefect Release Notes
 
+## Release 2.8.7
+
+If you have been watching the experimental section of our release notes, you may have noticed a lot of work around concurrency tooling and result artifacts. With this release, these experiments have culminated into exciting features.
+
+### Engine reliability
+
+Supporting mixed asynchronous and synchronous code is really complicated, but when designing Prefect 2 we wanted to account for the future growth of asynchronous Python and the many user requests for asynchronous task support. Most of this complexity is buried in the Prefect engine, which manages execution of your flows and tasks. With this release, we've made some drastic improvements to our engine closing some long-standing bugs and ensuring that the engine is not a point of failure when running your flows.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:9]:
```suggestion
Supporting mixed asynchronous and synchronous code is complicated, but important. When designing Prefect 2, we wanted to account for the future growth of asynchronous Python and the many user requests for asynchronous task support. Most of this complexity is buried in the Prefect engine, which manages execution of your flows and tasks. With this release, we've made some dramatic improvements to the engine, closing some long-standing bugs and ensuring that it isn't a point of failure when running your flows.
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,80 @@
...
+- https://github.com/PrefectHQ/prefect/pull/8887
+- https://github.com/PrefectHQ/prefect/pull/8903
+- https://github.com/PrefectHQ/prefect/pull/8830
+
+### Results page
+
+...
+
+### Enhancements
+- Add `--reverse` option to the flow run logs CLI to view logs in descending order  — https://github.com/PrefectHQ/prefect/pull/8625
```
Comment (billpalombi) [inline RELEASE-NOTES.md:53]:
```suggestion
- Add `--reverse` option to the flow run logs CLI to view logs in descending order — https://github.com/PrefectHQ/prefect/pull/8625
```

**Review by billpalombi — COMMENTED** [review]
Minor copyedits

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,93 @@
...
+- Create `emit_event` helper that takes args for an `Event` and emits it via a worker — https://github.com/PrefectHQ/prefect/pull/8867
+- Allow multiple artifacts to have the same key — https://github.com/PrefectHQ/prefect/pull/8855
+- Add common values to job configuration prior to flow run submission — https://github.com/PrefectHQ/prefect/pull/8826
+
+### Deprecations
+- Creating data documents will now throw deprecation warnings — https://github.com/PrefectHQ/prefect/pull/8760
+
+### Documentation
+- Add documentation for events and resources — https://github.com/PrefectHQ/prefect/pull/8858
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:86]:
```suggestion
- Add documentation for tracking of results - https://github.com/PrefectHQ/prefect/pull/8852

```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,94 @@
...
+- Flow timeouts are now enforced with signals
+    - Allows interrupt of long-running system calls like `sleep` for more effective timeout enforcement
+- Asynchronous flows can be called from sync flows
+- Asynchronous tasks can be used as upstream dependencies for sync tasks in async flows
+- Waiting for many tasks that sleep no longer causes deadlocks
+- Flows with thousands of synchronous tasks are less likely to crash
+- Debug mode now enables verbose logging from Prefect concurrency internals
+- The API limits itself to 100 concurrent requests when using SQLite as a backend
+    - Avoids database file contention when using high levels of concurrency
+- Resolving task inputs no longer users worker threads
```
Comment (serinamarie) [inline RELEASE-NOTES.md:26]:
```suggestion
- Resolving task inputs no longer uses worker threads
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,102 @@
...
+See the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect-ui-library/pull/1207
+- https://github.com/PrefectHQ/prefect-ui-library/pull/1213
+- https://github.com/PrefectHQ/prefect-ui-library/pull/1223
+- https://github.com/PrefectHQ/prefect/pull/8904
+- https://github.com/PrefectHQ/prefect/pull/8759
+
+### Flow run graph
+
+We heard that people loved the simplicity and sleekness of the timeline on the flow run page, but valued Radar's abaility to traverse between flow runs and subflows runs. This release introduces the ability to expand and collapse subflow runs within the timeline. The flow run timeline has now evolved into a general purpose flow run graph with the ability to render thousands of nodes and edged performantly and navigate your flow runs with ease.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:56]:
```suggestion
We heard that people loved the simplicity and sleekness of the timeline on the flow run page, but valued Radar's ability to traverse between flow runs and subflows runs. This release introduces the ability to expand and collapse subflow runs within the timeline. With these enhancements, the flow run timeline has now evolved into a general purpose flow run graph, with the ability to render thousands of nodes and edges performantly. You can now observe and explore your flow runs even more quickly and easily!
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,102 @@
...
+- Update to worker base job template logic for nested placeholders — https://github.com/PrefectHQ/prefect/pull/8795
+- Require lowercase artifact `key` field — https://github.com/PrefectHQ/prefect/pull/8860
+- Create `emit_event` helper that takes args for an `Event` and emits it via a worker — https://github.com/PrefectHQ/prefect/pull/8867
+- Allow multiple artifacts to have the same key — https://github.com/PrefectHQ/prefect/pull/8855
+- Add common values to job configuration prior to flow run submission — https://github.com/PrefectHQ/prefect/pull/8826
+
+### Deprecations
+- Creating data documents will now throw deprecation warnings — https://github.com/PrefectHQ/prefect/pull/8760
+
+Note: This release also removes the Radar flow run graph
```
Comment (madkinsz) [inline RELEASE-NOTES.md:91]: @billpalombi this feels pretty awkward. Let's just mention that it's been replaced in the feature highlight at the top and leave this out?

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,102 @@
...
+
+### Results tab on flow run pages
+
+The Prefect UI now renders information about your flow run and task run results! 
+
+This view provides a visual representation of the output of your tasks and flows and, when possible, provides links to results persisted using any of our storage blocks. To see this in your UI, run any flow and navigate to the run page; from there you'll see a new tab, "Results":
+
+![Screenshot 2023-03-23 at 12 37 47 PM](https://user-images.githubusercontent.com/27291717/227274576-1379c67c-6624-4a79-9bf7-83ae70e1fb4d.png)
+
+![Screenshot 2023-03-23 at 12 37 37 PM](https://user-images.githubusercontent.com/27291717/227274578-35673508-09e2-4b83-bc22-11538f813eea.png)
```
Comment (serinamarie) [inline RELEASE-NOTES.md:45]:
```suggestion
![results_list_view](https://user-images.githubusercontent.com/27291717/227274576-1379c67c-6624-4a79-9bf7-83ae70e1fb4d.png)

![result_grid_view](https://user-images.githubusercontent.com/27291717/227274578-35673508-09e2-4b83-bc22-11538f813eea.png)
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,102 @@
...
+See the following pull requests for implementation details:
+- https://github.com/PrefectHQ/prefect-ui-library/pull/1207
+- https://github.com/PrefectHQ/prefect-ui-library/pull/1213
+- https://github.com/PrefectHQ/prefect-ui-library/pull/1223
+- https://github.com/PrefectHQ/prefect/pull/8904
+- https://github.com/PrefectHQ/prefect/pull/8759
+
+### Flow run graph
+
+We heard that people loved the simplicity and sleekness of the timeline on the flow run page, but valued Radar's ability to traverse between flow runs and subflows runs. This release introduces the ability to expand and collapse subflow runs within the timeline. With these enhancements, the flow run timeline has now evolved into a general purpose flow run graph, with the ability to render thousands of nodes and edges performantly. You can now observe and explore your flow runs even more quickly and easily!
```
Comment (madkinsz) [inline RELEASE-NOTES.md:56]: Perhaps instead of "Radar's" have "the radar graph's"?

Comment (billpalombi) [reply RELEASE-NOTES.md:56]: Agreed thats better

File: RELEASE-NOTES.md (hunk 7 above)
Comment (billpalombi) [reply RELEASE-NOTES.md:91]: I'm sold

**Review by brandonreid — APPROVED** [review]

**Pull request merged**